.eslintrc

```diff
@@ -11,5 +11,31 @@
   "rules": {
     "no-console": 1, // Means warning
     "prettier/prettier": 2 // Means error
-  }
+  },
+  "overrides": [
+    {
+      "files": ["**/*.ts"],
+      "env": { "browser": true, "es6": true, "node": true },
+      "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended"
+      ],
+      "globals": { "Atomics": "readonly", "SharedArrayBuffer": "readonly" },
```

Comment:
Why do we have these globals?

Reply:
It is related to checks on the usage of globas. It is not needed.

Reply:
Removed